documentation/breaking-changes/upcoming-breaking-changes/Az.StorageSync-UpcomingBreakingChanges.md

```diff
@@ -2,27 +2,5 @@
 
  The following cmdlets were affected this release:
 
-
-
-
 ### **Invoke-AzStorageSyncFileRecall**
-The following parameters were affected this release:
```

Comment:
No need to update `documentation/breaking-changes/upcoming-breaking-changes/Az.StorageSync-UpcomingBreakingChanges.md`. Please revert it.

Reply:
Reverted it